src/OpenSpaceToolkit/Astrodynamics/EventCondition.cpp

```diff
@@ -29,11 +75,42 @@ String EventCondition::getName() const
     return name_;
 }
 
+std::function<Real(const State&)> EventCondition::getEvaluator() const
+{
+    return evaluator_;
+}
+
+EventCondition::Target EventCondition::getTarget() const
+{
+    return target_;
+}
+
+void EventCondition::updateTarget(const State& aState)
+{
+    if (target_.type_ == EventCondition::Target::Type::Absolute)
+    {
+        throw ostk::core::error::RuntimeError("Can only set state for 'Relative' conditions.");
+    }
+
+    if (!evaluator_)
+    {
+        throw ostk::core::error::runtime::Undefined("Evaluator.");
```

Comment:
```suggestion
        throw ostk::core::error::runtime::Undefined("Evaluator");
```

nit, because this will print as `Evaluator. is undefined`